csharp/samples/SampleSender/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace SampleSender
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.EventHubs;
+
+    public class Program
+    {
+        private const string EhConnectionString = "{Event Hubs connection string}";
+        private const string EhEntityPath = "{Event Hub path/name}";
+
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Press Ctrl-C to stop the sender process");
+            Console.WriteLine("Press Enter to start now");
+            Console.ReadLine();
+
+            // GetAwaiter().GetResult() will avoid System.AggregateException
+            SendMessageToEventHubs().GetAwaiter().GetResult();
+        }
+
+        private static async Task SendMessageToEventHubs()
+        {
+            var connectionSettings = new EventHubsConnectionSettings(EhConnectionString)
+            {
+                EntityPath = EhEntityPath
+            };
+
+            var eventHubClient = EventHubClient.Create(connectionSettings);
+            while (true)
+            {
+                try
+                {
+                    var message = Guid.NewGuid().ToString();
```

Comment:
>var message = Guid.NewGuid().ToString(); [](start = 20, length = 40)

Let's send a message with readable text like "This is message 124". This would be easier to track the messages on the receiver side.